packages/xod-client/src/editor/containers/Patch.jsx

```diff
@@ -95,21 +118,25 @@ class Patch extends React.Component {
   }
 
   onCommentResizeHandleMouseDown(event, commentId) {
+    if (isMiddleButtonPressed(event) || this.props.mode === EDITOR_MODE.PANNING) return;
```

Comment:
Yep. IMO, the cleanest way to deal with this is to make Patch component act like a state machine that delegates event handling/actual rendering to different components depending on editor mode. But this had severe performance impact, so for now i resorted to this spaghetti code 🙈